Makefile

```diff
@@ -27,6 +27,10 @@ ifneq ($(USE_PTRGUARD),)
 	BUILD_TAGS := $(BUILD_TAGS),ptrguard
 endif
 
+ifneq ($(USE_CACHE),)
```

Comment:
USE_CACHE seems a bit general, it could get confusing if we supported other kinds of caching for the tests. How about USE_GO_CACHE or USE_CACHE_VOLUME, something along those lines?

Reply:
That was intentional, because I though if we have other caching later, we want to switch it with the same switch. What caching are you thinking of, that we might want to enable separately?

Reply:
OK, that's fine. I didn't have anything specific in mind, just a vague suspicion. But if we did have other kinds of caching one could enable I'm fine with having a less granular USE_CACHE as high level option and if we needed more granualar controls for each kind of caching we could add them later.